tasks/terraform-cli/src/context/azdo-task-context.ts

```diff
@@ -133,6 +133,9 @@ export default class AzdoTaskContext implements ITaskContext {
     get workspaceName() {
         return this.getInput("workspaceName", true);
     }
+    get skipExistingWorkspace() {

```

Comment:
@loispostula this also needs to be added/defined in the [`task.json`](https://github.com/charleszipp/azure-pipelines-tasks-terraform/blob/main/tasks/terraform-cli/task.json) file or it will not be possible to define it within your pipeline configuration. 

I would suggest adding the following which will make this optional and default to the current behavior (false).

```json

        {
            "name": "skipExistingWorkspace",
            "type": "boolean",
            "label": "Skip Create If Exists",
            "defaultValue": "false",
            "required": false,
            "helpMarkDown": "Skip attempting to create existing workspaces",
            "visibleRule": "workspaceSubCommand = new"
        }
```